frontend/src/metabase/static-viz/components/XYChart/utils/ticks.ts

```diff
@@ -159,6 +159,14 @@ export const getYTickWidths = (
   };
 };
 
+/**
```

Comment:
@JesseSDevaney I added the comment to the whole function instead, since this is the reason this function exists in the first place, not just the new logic we just added in this PR.